.github/workflows/test.yml

```
@@ -8,13 +8,18 @@ on:
 jobs:
   prepack:
     runs-on: ubuntu-18.04
-    name: Prepack
+    name: prepack
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/cache@v1
+      - name: cache node_modules
+        uses: actions/cache@v2
+        # cache node_modules based on lockfile which can change
+        # when there is no explicit change in package.json
+        # and prevents cache bust when scripts or metadata is changed
+        # which does not effect the node_modules themselves
         with:
-          path: /home/runner/.cache/yarn/v6
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
```

I like to have a key version as well which lets us just arbitrarily blow away the cache by swapping out keys in the event that we want to change our caching strategy without actually changing our `yarn.lock`, e.g.

```suggestion
          key: cachekey-v1-${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
```

---

entirely reasonable ❤️